site/class-1-load/jajan20/index.JSON

```diff
@@ -0,0 +1,18 @@
+[
```

Comment:
This file should be called `index.json`, not `index.JSON`.

You can rename files through the github interface, by going to your fork, your branch, and the finding your file. Finally, click on the file-name to edit it.